argus/backend/plugins/sct/service.py

```diff
@@ -63,6 +63,8 @@ def submit_packages(run_id: str, packages: list[dict]) -> str:
             run: SCTTestRun = SCTTestRun.get(id=run_id)
             for package_dict in packages:
                 package = PackageVersion(**package_dict)
+                if "target" in package.name:
+                    SCTService.process_target_version(run, package)
```

Comment:
we have scylla-server-target and scylla-server-upgrade-target packages. Depending which is last in the list it will decide - is it good enough?

Reply:
So that's handled by `process_target_version`, the `upgrade-target` should always take priority, so the function checks if `version_source` has already been set to `upgrade-target` and denies `scylla-server-target` the update to the version. This should handle the submission in arbitrary order.